lms/djangoapps/lms_migration/management/commands/create_user.py

```diff
@@ -44,7 +44,7 @@ def complete(self, text, state):
 
 
 def GenPasswd(length=8, chars=string.letters + string.digits):
-    return ''.join([choice(chars) for i in range(length)])
+    return ''.join([choice(chars) for dummy0 in range(length)])
```

Comment:
Per above, I contend that `dummyN` is more transparent and to be preferred over `_` and `__`.